edge-modules/edgehub-proxy/src/error.rs

```diff
@@ -1,113 +1,59 @@
 // Copyright (c) Microsoft. All rights reserved.
 
-use std::fmt::{self, Display};
+//use std::fmt::{self, Display};
```

Comment:
remove the commented line?

Reply:
Addressed this in an earlier resolved comment. Resolving this as well

Reply:
I still see the commented out code